apps/chat-e2e/src/tests/promptFolderNumerationTest.test.ts

```diff
@@ -0,0 +1,297 @@
+import { Prompt } from '@/chat/types/prompt';
+import dialTest from '@/src/core/dialFixtures';
+import {
+  ExpectedConstants,
+  ExpectedMessages,
+  MenuOptions,
+} from '@/src/testData';
+import { expect } from '@playwright/test';
+
+dialTest(
+  'Prompt folder: default numeration',
+  async ({
+    dialHomePage,
+    promptBar,
+    folderPrompts,
+    setTestIds,
+  }) => {
+    setTestIds('EPMRTC-1621');
+
+    await dialTest.step(
+      'Create several new prompt folders and verify their names are incremented',
+      async () => {
+        await dialHomePage.openHomePage();
+        await dialHomePage.waitForPageLoaded();
+        for (let i = 1; i <= 3; i++) {
+          await promptBar.createNewFolder();
+          await expect
+            .soft(
+              folderPrompts.getFolderByName(
+                ExpectedConstants.newPromptFolderWithIndexTitle(i),
```

Comment:
autogenerated folder name is the same on both side panels, so you may use: `ExpectedConstants.newFolderWithIndexTitle(i)`

Reply:
fixed

Reply:
Not going to be fixed, because the behavior can change in the future and it will be easier to change the constant then to look for it in tests

Reply:
ok